helm/prometheus-agent/charts/prometheus-agent/values.yaml

```diff
@@ -86,6 +86,31 @@ serviceMonitorSelector: {}
 podMonitorNamespaceSelector: {}
 podMonitorSelector: {}
 
+# scrapeClasses allows to define relabelings for specific scrape classes that can be used in ServiceMonitors and PodMonitors.
+# More info: https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#monitoring.coreos.com/v1.ScrapeClass
+scrapeClasses:
+  - name: giantswarm-servicemonitors
+    default: false
```

Comment:
Do we need to set `default: true` to activate that scrape class by default on all our SM ?
But if we do it, isn't it a risk that we relabel customer SM also ?

Reply:
I would leave it to false yes :)

Reply:
So, we have a "not fun" task which is to update all ServiceMonitors and PodMonitors in all GS repositories...

Reply:
That's a fun task if you make it a script :)